### DISCUSSION TITLE
How do I change this icon?

### DISCUSSION THREAD

**Discussion by Evgeni1223** [TomSchimansky/CustomTkinter#1777] [General]
![w](https://github.com/TomSchimansky/CustomTkinter/assets/132823368/871937ae-aec9-47f0-a662-61a43e750c03)

**Comment by YoussefEssalhi** [comment]
With **iconbitmap** method :

```
app.iconbitmap("ICON_PATH")
```
-> Result :

![change_icon_customtkinter](https://github.com/TomSchimansky/CustomTkinter/assets/76540840/3aea0dfb-6031-4389-9c09-a63a660de9bb)

* Make sure the icon extension is ".ico" and that the icon is located in the same folder.

**Reply by Evgeni1223** [reply]
Thanks
